README.md

```diff
@@ -418,6 +418,15 @@ configuration: /etc/postfix-mailman.cfg
 
 The configuration file `/etc/postfix-mailman.cfg` is generated automatically.
 
+# Setup site owner address. By default, mailman is setup with the site_owner set to 'changeme@example.com'. This should be pointing to a valid mailbox. Add the following to the '/opt/mailman/core/mailman-extra.cfg'.
```

Comment:
Can you wrap the changes in triple backticks?

```
Something like this
```
So it looks like a "code". By default, in Markdown, `#` is a H1 and not comment.

Reply:
Ok. I believe I have right now.

Reply:
Looks great! Thanks!